build.rs

```diff
@@ -17,6 +17,7 @@ fn main() -> Result<()> {
         let path = path.to_string_lossy();
         println!("cargo:rerun-if-changed={}", path);
     }
+    println!("cargo:rerun-if-changed=./libs/x86_64/linux/musl/libkflow.a");
```

Comment:
question: Was this purely for debugging or meant to stay?

Reply:
This was meant to stay. 
Otherwise, updating the libkflow library won't trigger a rebuild. Which could cause a variety of caching problems.